test/confit-test.js

```diff
@@ -26,6 +26,10 @@ test('confit', function (t) {
 
 
     t.test('get', function (t) {
+        //setting process.env.env to development, should not change 'env:env'.
+        //This should be ignored and 'env:env' should solely depend on process.env.NODE_ENV
+        process.env.env = 'development';
```

Comment:
Cool. Like I said, I didn't run it locally. But as long as that's the case all is well!